README.md

```diff
@@ -245,6 +245,7 @@ This is only required on Cloud TPU VMs.
 | Version | Cloud TPU VMs Docker |
 | --- | ----------- |
 | 2.3 | `us-central1-docker.pkg.dev/tpu-pytorch-releases/docker/xla:r2.3.0_3.10_tpuvm` |
+| 2.2 | `us-central1-docker.pkg.dev/tpu-pytorch-releases/docker/xla:r2.2.0_3.10_tpuvm` |
```

Comment:
can we make sure these docker actually exists before merging?

Reply:
Cannot verify the 2.3 since it's not built yet, however, we need to update the readme before the final build. See title for reason.

Reply:
@will-cromar Thanks for the heads up, just removed